src/main/java/org/jitsi/jicofo/ChatRoomRoleAndPresence.java

```diff
@@ -101,7 +101,8 @@ public ChatRoomRoleAndPresence(JitsiMeetConferenceImpl conference,
      */
     public void init()
     {
-        authAuthority = Main.getJicofoServices().getAuthenticationAuthority();
+        authAuthority = JicofoServices.jicofoServicesSingleton == null
```

Comment:
Is this only true in the unit test case?  When unit testing, don't we need to stub out `JicofoServices.jicofoServicesSingleton` ?

Reply:
We could, but currently we don't (this is the only place that uses it). Let's see how this evolves.